packages/mobile/src/identity/contactMapping.test.ts

```diff
@@ -39,7 +38,6 @@ describe('Import Contacts Saga', () => {
     await expectSaga(doImportContacts)
       .provide([
         [call(getConnectedAccount), null],
-        [call(waitForUserVerified), true],
```

Comment:
It's unrelated. it was only there because of the saga interference bug, which was mitigated when we moved import into a separate onboarding screen. So this is dead code now.